lib/plugins/aws/deployList/index.js

```diff
@@ -14,98 +13,96 @@ class AwsDeployList {
     Object.assign(this, validate, setBucketName);
 
     this.hooks = {
-      'before:deploy:list:log': () => BbPromise.bind(this).then(this.validate),
-      'before:deploy:list:functions:log': () => BbPromise.bind(this).then(this.validate),
-
-      'deploy:list:log': () =>
-        BbPromise.bind(this)
-          .then(this.setBucketName)
-          .then(this.listDeployments),
-      'deploy:list:functions:log': () => BbPromise.bind(this).then(this.listFunctions),
+      'before:deploy:list:log': async () => this.validate(),
```

Comment:
In single method calls we may skip `async` token. Sometimes it's a _sync_ function that's invoked, and hooks accept both _async_ and _sync_ callbacks